sdk/genesis-config/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "solana-genesis-config"
+description = "The chain's genesis config."
+documentation = "https://docs.rs/solana-genesis-config"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+bincode = { workspace = true }
+chrono = { workspace = true }
+memmap2 = { workspace = true }
```

Comment:
It looks like these deps are only used by genesis config -- can you remove them from solana-sdk?